### PR TITLE
MQTT session disconnect UI and service startup ordering changes

### PR DESCRIPTION
**Pull request by richturner**

### CODE REVIEW

**Comment by richturner** [comment]
Includes some logging updates and a fix for gradle build handling of encrypted files

**Comment by richturner** [comment]
This branch is now deployed on https://test2.openremote.app

File: ui/app/manager/src/pages/page-users.ts [hunk 1]
```diff
@@ -735,30 +750,33 @@ export class PageUsers extends Page<AppStateKeyed> {
...
+    protected async getSessionLoader(user: UserModel): Promise<TemplateResult> {
+        const userSessionsResponse = await (manager.rest.api.UserResource.getUserSessions(manager.displayRealm, user.id));
+
+        if (!this.responseAndStateOK(() => userSessionsResponse.status === 200, userSessionsResponse, i18next.t("loadFailedUserInfo"))) {
+            return html``;
+        }
+
+        const cols = [i18next.t("address"), i18next.t("since"), ""];
+        const rows = userSessionsResponse.data.map((session) => {
+            return [session.remoteAddress, new Date(session.startTimeMillis), html`<or-mwc-input raised .type="${InputType.BUTTON}" label="${i18next.t("disconnect")}" @or-mwc-input-changed="${() => {this.disconnectSession(user, session)}}"></or-mwc-input>`]
```
Comment (DonWillems) [inline ui/app/manager/src/pages/page-users.ts:774]: I think it is better to remove 'raised'. The primary button of the page is the 'save' button.

**Review by DonWillems — CHANGES_REQUESTED** [review]
Minor styling request for the button.

File: ui/app/manager/src/pages/page-users.ts (hunk 1 above)
Comment (richturner) [reply ui/app/manager/src/pages/page-users.ts]: Done

**Review by DonWillems — APPROVED** [review]

**Pull request merged**